src/app/shared/modal/modal-login/modal-login.component.ts

```diff
@@ -30,16 +35,14 @@ export class ModalLoginComponent implements OnInit {
   }
 
   login(): void {
-    const newUser: User = {
-      id: this.id,
-      email: this.email,
-      password: this.password
-    };
-    this.authService.login(newUser);
-    // this.userLogged.emit();
+    this.courseService.loadingSubject.next(true);
```

Comment:
Please create the separate service for the handling of the loading logic and use it in the service with the help of the **RxJS**

Reply:
ok, made with separate LoadingService